test/functional/mongo_client_tests.js

```diff
@@ -420,13 +420,12 @@ describe('MongoClient', function() {
       var configuration = this.configuration;
       var MongoClient = configuration.require.MongoClient;
 
-      try {
-        MongoClient.connect('user:password@localhost:27017/test', function(err, client) {
-          client.close();
-        });
-      } catch (err) {
+      MongoClient.connect('user:password@localhost:27017/test', function(err, client) {
+        test.notEqual(err, null);
+        test.notEqual(err, undefined);
         done();
-      }
+        client.close();
```

Comment:
Should `client.close();` happen before `done()`?

Reply:
Oops, yes, had that in mind and then lost it 👍

Reply:
Good catch cause I had that test re-worked incorrectly expecting no error instead of error 😯